README.md

```diff
@@ -20,23 +20,42 @@ Source code annotation tool offers an UI to annotate source code and review thes
 
 ## Installation
 
-## Github OAuth tokens
+## Environment Variables
 
-1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+The next sections make use of several environment variables to configure the application. In this table you will find all of them grouped as a quick reference:
 
-In order to be able to use this application while running the tool locally, make sure you add http://127.0.0.1:8080/oauth-callback to the authorization callback URL field.
+| Variable | Required | Default value | Meaning |
```

Comment:
people often move not mandatory variables somewhere in the bottom of README. Currently, the table is larger than my display and we are going to have more... Just as an idea for improvement.

Reply:
Since this `README.md` is already huge and imho it does not strictly follow our [recommendations for `README.md`](https://github.com/src-d/guide/blob/master/engineering/documentation.md#readmemd) I would not care too much about the length of the table: I'd merge as soon as possible, and improve `README.md` when facing issue  https://github.com/src-d/code-annotation/issues/67

I'd maybe reorder the mandatory ones in a way it appears first, but it's up to you @carlosms

Reply:
Thanks both for the feedback, I moved the mandatory vars to the top of the table.